inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/RdfUtils.java

```diff
@@ -77,12 +90,27 @@
             throws QueryEvaluationException {
         return conn.getStatements(subj, pred, obj, includeInferred);
     }
-    
+
     public static RepositoryResult<Statement> getStatementsSparql(RepositoryConnection conn,
-            Resource subj, IRI pred, Value obj, boolean includeInferred)
+        Resource subj, IRI pred, Value obj, boolean includeInferred)
+        throws QueryEvaluationException
+    {
+        return getStatementsSparql(conn, subj, pred, obj, includeInferred, null);
+    }
+
+    public static RepositoryResult<Statement> getStatementsSparql(RepositoryConnection conn,
+            Resource subj, IRI pred, Value obj, boolean includeInferred, String language)
         throws QueryEvaluationException
     {
-        String QUERY = "SELECT * WHERE { ?s ?p ?o } LIMIT 1000";
+        String filter = "";
+        if (language != null) {
+            filter = "FILTER(LANG(?o) = \"\" || LANGMATCHES(LANG(?o), \"" + language + "\")).";
```

Comment:
Can the user could potentially set any language in the KB config and inject it here? We do not escape it, so I am wondering.